routers/web/web.go

```diff
@@ -1045,8 +1045,9 @@ func registerRoutes(m *web.Router) {
 		m.Group("/migrate", func() {
 			m.Get("/status", repo.MigrateStatus)
 		})
+		m.Get("/issues/suggestions", repo.IssueSuggestions)
```

Comment:
How about put the router under `/{username}/{reponame}/issues/suggestions` or ``/{username}/{reponame}/issues/-/suggestions`?

Reply:
I thought using `/-/` inside would make sense as it is an endpoint returning json instead of html.

Reply:
No such rule. And many routers with `/-/` will not return JSON.
Another problem is the permission is `reqRepoCodeReader`, I think it should be `reqRepoIssuesOrPullsReader` and the backend should also check the permissions to display `issues only`, `pull request only` or `both issues and pull requests`.